forest/tree/tree.go

```diff
@@ -0,0 +1,47 @@
+package tree
+
+import (
+	cache "github.com/treeverse/lakefs/forest/cache_map"
+	gr "github.com/treeverse/lakefs/graveler"
+	"github.com/treeverse/lakefs/graveler/committed/sstable"
+)
+
+type TreePart struct {
+	PartName sstable.ID `json:"part_name"`
+	MaxKey   gr.Key     `json:"max_path"`
+}
+type TreeSlice struct {
+	treeSlice []TreePart
+}
+
+type treesRepo struct {
```

Comment:
Change "trees-repo" to tree-repo (singular) in all relevant places

Reply:
after commit